modules/apps/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/search_container_select.js

```diff
@@ -120,10 +123,11 @@ AUI.add(
 
 					var elements = [];
 
-					var selectedElements = instance.getAllSelectedElements();
+					var allElements = instance._getAllElements(false);
```

Comment:
Hi @ChrisKian could you be more specific in naming your variables. `allElements` is very generic, you could name it something like `inputElements`, or whatever they represent.

Reply:
Hi @kresimir-coko,
I use `allElements` because the variable is gathered from the `instance._getAllElements()` method.  I can still change it if you'd like.

Reply:
So `allElements` contains all of the elements inside its parent? Or does it return some specific subset of elements? Like all `<select>` elements, etc.?

Reply:
In this case, it's going to return all checkbox elements, because we are dealing with the search container _select_ option.

And the clarify, it will return all checkboxes which have been rendered.  For example, if we only load the first page and do not paginate, `instance._getAllElements(false)` will only return the first page's checkboxes.